### PR TITLE
Feature/modern js

### PR DESCRIPTION
**Pull request by andrew72ru**
## Description

* Fix upload button position in "Add media" page;
* Refactor js for better performance and control;
* Use Uploadcare Image Editor for modify remote images;

## Checklist

- [x] Changelog stub (or use [conventional commit messages](https://www.conventionalcommits.org/))

### CODE REVIEW

File: jsx/admin.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import './uc-picker-wrapper.scss';
+import UcEditor from './imageEdit/editorLoader';
+import UploadToLibrary from './UploadToLibrary';
+
+const cdnRegex = new RegExp('ucarecdn.com');
```
Comment (nd0ut) [inline jsx/admin.js:5]: Не совсем понимаю что это и где это в контексте вордпресса, но хардкод `ucarecdn.com` здесь допустим? Не надо ли использовать cdnBase из конфига?

Comment (andrew72ru) [reply jsx/admin.js:5]: Of course we should. It will be fixed soon, thank you.

File: admin/UcAdmin.php [hunk 2]
```diff
@@ -125,37 +133,62 @@ public function add_uploadcare_js_to_admin($hook)
...
         ];
 
         echo \json_encode($result);
         \wp_die();
     }
 
+    public function loadPostByUuid(string $uuid): ?\WP_Post
+    {
+        global $wpdb;
+        $query = \sprintf('SELECT post_id FROM `%s` WHERE meta_value=\'%s\' AND meta_key=\'uploadcare_uuid\'', \sprintf('%spostmeta', $wpdb->prefix), $uuid);
```
Comment (nd0ut) [inline admin/UcAdmin.php:162]: I'm not sure, but seems that query isn't being escaped. Perhaps `$uuid` value cannot be affected by user, but it's better to escape it IMHO.

**Comment by andrew72ru** [comment]
All issues fixed

**Pull request merged**